lib/jekyll.rb

```diff
@@ -78,7 +78,7 @@ class << self
     # images and allows you to skip that when working in development.
 
     def env
-      ENV["JEKYLL_ENV"] || "development"
+      ENV["JEKYLL_ENV"] || "local"
```

Comment:
I dislike this particular change. `local` doesn't have as heavy of a semantic meaning as `development` does. `development` is also seen in other places, such as `RACK_ENV` and `RAILS_ENV` and I think keeping that consistency is important.